package.json

```diff
@@ -24,7 +24,7 @@
     "launchdarkly-js-client-sdk": "^2.18.0",
     "lodash": "^4.17.20",
     "regenerator-runtime": "^0.13.7",
-    "sbc-common-components": "2.4.18",
+    "sbc-common-components": "2.4.19",
```

Comment:
Please explicitly identify the changes between 18 and 19 in the tickets for QA to verify.

Reply:
done.

Reply:
Where is this documented?

The problem is that it's hard to tell what's new in a new version of sbc-common-components (so that we know what to test or watch for).

The solution is to explicitly document, in the ticket, what changes exist between the previous version and the new version we are importing.

Reply:
You may need to look at the PR's comment of sbc-common-components.

Reply:
^^ Yes, that is what needs to be done.

Since you are the dev updating the sbc-common-components dependency (and since there is no other way to keep track of the changes between versions), it falls to you to document the changes, in the ticket, for QA to see (please!).

(I would love another way to know about changes -- say, [a release bulletin](https://github.com/bcgov/sbc-common-components/releases) -- so that we could just refer to that instead of having to dive into the commit history every time.) << @sumesh-aot @saravankumarpa FYI

Reply:
Ok, I may put the change detail into bcgov/entity#6801. Which you can include this ticket into your release plan. Is that ok?

Reply:
Thanks. Will do!